src/App.css

```diff
@@ -421,6 +421,8 @@ input[type='radio'][disabled] {
 /*fix: select makes container to grow*/
 select {
   width: 100%;
```

Comment:
This fix the fact that the down arrow doesnot display on chrome as it is hidden by templete.dat.gouv.

I added the fix here as there was already a fix of template.data.gouv's CSS. I should have added a comment.

I tried to fix the select 100% width broken behaviour but didnot succeed. 

I think there should be a dedicated dropdown component in atoms. But I believe this PR is not the right place for it.